CHANGELOG.rst

```diff
@@ -5,6 +5,11 @@ Version History
 v5.14.8
```

Comment:
I'm approving this but the Changelog needs to be updated to v5.14.9
Thanks for taking care of this!